specification/liftrastronomer/Astronomer.Astro.Management/main.tsp

```diff
@@ -32,6 +32,18 @@ enum Versions {
   @useDependency(LiftrBase.Versions.v1_preview)
   @useDependency(LiftrBase.Data.Versions.v1_preview)
   v1: "2023-08-01",
+
+  @doc("Dependent on Azure.ResourceManager.Versions.v1_0_Preview_1, LiftrBase.Versions.v2024_08_27_preview, LiftrBase.Data.Versions.v1_preview")
+  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+  @useDependency(LiftrBase.Versions.v2024_08_27_preview)
+  @useDependency(LiftrBase.Data.Versions.v1_preview)
+  v2024_08_27_preview: "2024-08-27-preview",
+
+  @doc("Dependent on Azure.ResourceManager.Versions.v1_0_Preview_1, LiftrBase.Versions.v2024_08_27_preview, LiftrBase.Data.Versions.v1_preview")
+  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+  @useDependency(LiftrBase.Versions.v2024_08_27_preview)
```

Comment:
curios. shouldn't these be the stable version instead of preview?

Reply:
We can add stable version separately and then use it here. But in our case both preview and stable version are exactly same, so we are reusing it instead of defining versions twice (for preview and stable both) at every place.

Reply:
At the core, we have preview version only (azure-core.tsp).
```
...
enum Versions {
  @doc("Version 1.0-preview.1")
  v1_0_Preview_1: "1.0-preview.1",

  @doc("Version 1.0-preview.2")
  v1_0_Preview_2: "1.0-preview.2",
}
...
```